curricula/views.py

```diff
@@ -248,14 +248,18 @@ def lesson_view(request, slug, unit_slug, lesson_num, optional_num=False):
 
     changelog = Version.objects.get_for_object(lesson).filter(revision__user__username=settings.CHANGELOG_USER)
 
+    # disable admin controls if user cannot edit inline
+    can_administer = lesson.is_editable(request)
```

Comment:
pass this in because templates can't pass arguments to methods: https://docs.djangoproject.com/en/1.8/ref/templates/language/#accessing-method-calls